_includes/methods/print-header.html

```diff
@@ -0,0 +1,30 @@
+<div class="card-header no-print">
+  <nav>
+
+      {% if layout == "method-category" %}
+      <a href="{{ '/methods' | url }}/" class="usa-button usa-button--secondary">
+        {% image_with_class "assets/methods/img/cards.png", "methods--cards-img", "" %}
+        See all cards
+      </a>
+      {% else %}
+      <a href="{{ '/methods' | url }}/{{ page.category | downcase}}" class="usa-button usa-button--secondary margin-bottom-1" onclick="ga('send', 'event', 'Navigate', 'Click method category link','Navigate to method categoy page');">
```

Comment:
[question] I'm not particularly familiar with google analytics, though I'm assuming that's what `ga` is referring to, right?  What is the importance of this onclick event?

Reply:
Great question, I don't know the answer yet. I have an [outstanding issue](https://github.com/18F/guides/issues/235) in this epic to finalize the printing functionality, so I'll link this question there.